backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/cfg/ElasticsearchBackendSettings.java

```diff
@@ -28,40 +25,35 @@
  *
  * @author Gunnar Morling
  */
-@HibernateSearchConfiguration(
-		prefix = { "hibernate.search.backend.", "hibernate.search.backends.<backend name>." },
-		title = "Hibernate Search Backend - Elasticsearch",
-		anchorPrefix = "hibernate-search-backend-elasticsearch-"
-)
 public final class ElasticsearchBackendSettings {
 
 	private ElasticsearchBackendSettings() {
 	}
 
 	/**
-	 * The name to use for the {@link org.hibernate.search.engine.cfg.BackendSettings#TYPE backend type}
-	 * configuration property so that an Elasticsearch backend is instantiated by Hibernate Search.
+	 * The value to set the {@link org.hibernate.search.engine.cfg.BackendSettings#TYPE backend type}
+	 * configuration property to
+	 * in order to get an Elasticsearch backend instantiated by Hibernate Search.
 	 * <p>
 	 * Only useful if you have more than one backend technology in the classpath;
 	 * otherwise the backend type is automatically detected.
 	 */
-	@HibernateSearchConfiguration(ignore = true)
 	public static final String TYPE_NAME = "elasticsearch";
 
 	/**
-	 * The host name and ports of the Elasticsearch servers to connect to.
-	 * <p>
-	 * Expects a String representing a host and port such as {@code localhost} or {@code es.mycompany.com:4400},
-	 * or a String containing multiple such host-and-port strings separated by commas,
-	 * or a {@code Collection<String>} containing such host-and-port strings.
+	 * The hostname and ports of the Elasticsearch servers to connect to.
 	 * <p>
-	 * Cannot be used if {@link #URIS} is set.
-	 * <p>
-	 * Defaults to {@link Defaults#HOSTS}.
+	 * Expects a String representing a hostname and port such as {@code localhost} or {@code es.mycompany.com:4400},
+	 * or a String containing multiple such hostname-and-port strings separated by commas,
+	 * or a {@code Collection<String>} containing such hostname-and-port strings.
 	 * <p>
 	 * Multiple servers may be specified for load-balancing: requests will be assigned to each host in turns.
 	 * <p>
+	 * This property cannot be used if {@link #URIS} is set.
```

Comment:
Maybe it would be better to say that it causes an exception. (and the same with the protocol/uris below)